src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -19,35 +21,42 @@ public class TestJobs2dPatterns {
 
 	/**
 	 * Setup test concerning preset figures in context.
-	 * 
+	 *
 	 * @param application Application context.
 	 */
 	private static void setupPresetTests(Application application) {
 		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
 				DriverFeature.getDriverManager());
 
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
+		application.addTest("Figure Joe 2", selectTestFigureOptionListener);
+		application.addTest("Figure Jane 1", selectTestFigureOptionListener);
+		application.addTest("Figure Rectangle 1", selectTestFigureOptionListener);
+		application.addTest("Figure Triangle 1", selectTestFigureOptionListener);
```

Comment:
Figure enums should be used also here. Changing only one leads to an error (inconsistency).